src/pybind/mgr/volumes/module.py

```diff
@@ -165,6 +165,16 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
                     "and optionally, in a specific subvolume group",
             'perm': 'rw'
         },
+        {
+            'cmd': 'fs subvolume resize '
+                   'name=vol_name,type=CephString '
+                   'name=sub_name,type=CephString '
+                   'name=new_size,type=CephInt,req=true '
```

Comment:
> @batrick : Created https://tracker.ceph.com/issues/42479.

TBH, I think it would better to just move the command into `resize`: change `new_size` type to `CephString`. Try to parse it as an int (or file size, I think we have methods in the mgr for this!) and then try to parse it as "infinite".

Reply:
(To be clear: that can still go in a followup PR. It doesn't need to be in this one.)

Reply:
okay, let me try that way first as a followup PR.